modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -16,8 +16,7 @@ The Data Mover solution uses the Restic option of VolSync.
 
 Data Mover supports backup and restore of CSI volume snapshots only.
 
-In OADP 1.2 Data Mover `VolumeSnapshotBackups` (VSBs) and `VolumeSnapshotRestores` (VSRs) are queued using the VolumeSnapshotMover (VSM). The VSM's performance is improved by specifying a concurrent number of VSBs and VSRs simultaneously `InProgress`. After all async plugin operations are complete, the backup is marked as complete.
-
+In OADP 1.2 Data Mover, `VolumeSnapshotBackups` (VSBs) and `VolumeSnapshotRestores` (VSRs) are queued using the VolumeSnapshotMover (VSM). The VSM's performance is improved by specifying a concurrent number of VSBs and VSRs simultaneously `InProgress`.
```

Comment:
This change is more than just a typo. I think we need a QE approval here?

Reply:
@snarayan-redhat The sentence involved is no doubt true, but it has nothing to do with performance improvement, which is the subject of the paragraph. I will leave it in for now so we can fix the typo in the document.